server/config/forms/incidentDetailsForm.js

```diff
@@ -65,7 +59,9 @@ const requiredIncidentDate = joi
 const transientSchema = joi.object({
   incidentDate: requiredIncidentDate.meta({ fieldType: EXTRACTED }).alter(optionalForPartialValidation),
 
-  locationId: requiredIntegerMsg('Select the location of the incident').alter(optionalForPartialValidation),
```

Comment:
Both incidentLocationId and locationId are needed in the validation schema because the /incident-details page encountered when making a new report requires  incidentLocationId to be selected. However in /check-your-answers, we need to ensure validation passes where a user may be on the page when this new code change is deployed and hence may only have the locationId persisted to db. Not having the locationId in the validation schema will cause validation to fail even though it previously passed and will cause the app to fail when user submits the form